### PR TITLE
Implement basic navigation bar and Routing system for the app

### PR DESCRIPTION
**Pull request by karolkosek**

### CODE REVIEW

**Comment by karolkosek** [comment]
Screenshot 2022-12-14 at 13 52 26

File: tsconfig.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
-    "jsx": "react-jsx"
+    "jsx": "react-jsx",
```
Comment (szymonborda) [inline tsconfig.json:21]: "As JSON is based on a very restricted subset of JavaScript syntax, trailing commas are not allowed in JSON."
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas

File: src/components/AppLayout/Sidebar/Sidebar.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+const Sidebar = ({ children }: SidebarProps) => {
+  return (
+    <Box sx={{ width: "256px", height: "100vh", 
+          padding: "1rem", boxSizing: "border-box", 
+          border: "1px solid rgba(0, 0, 0, 0.12)",
+          backgroundColor: "#FCFBFA"}}>
+        <Box sx={{ display: "flex", gap: "0.25rem"}} component="header">
+          <img src="/north-pole-factory-logo-icon.svg" alt="Blue star" />
+          <img src="/north-pole-factory-logo-text.svg" alt="North Pole Factory" />
+        </Box>
```
Comment (szymonborda) [inline src/components/AppLayout/Sidebar/Sidebar.tsx:16]: to mogłoby być linkiem do `/`

File: public/index.html [hunk 3]
```diff
@@ -2,14 +2,12 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
```
Comment (szymonborda) [inline public/index.html:5]: można zrobić jakiś favicon na szybko generatorem + zmienić `<title>`  i `<meta
      name="description">` poniżej

Comment (karolkosek) [reply public/index.html:5]: done

**Review by Fadikk367 — APPROVED** [review]

**Pull request merged**